docs/tables/config_ini.md

```diff
@@ -0,0 +1,121 @@
+# Table: config_ini
+
+Query data from INI files. The table will store all configurations along with sections of all INI file found in the configured `paths`.
+
+For instance, if `paths` is set to `/Users/myuser/ini/*`, and that directory contains:
+
+- defaults.ini
+- sample.ini
+
+This table will store all the configurations from each file mentioned above, along with section details, and comments.
+
+Which you can then query directly:
+
+```sql
+select
+  section,
+  key,
+  value
+from
+  config_ini;
+```
+
```

Comment:
Can you please include the results of what these queries produce as well in another code block?